examples/angularjs/js/controllers/todoCtrl.js

```diff
@@ -23,10 +23,9 @@ angular.module('todomvc')
 		// Monitor the current route for changes and adjust the filter accordingly.
 		$scope.$on('$routeChangeSuccess', function () {
 			var status = $scope.status = $routeParams.status || '';
-
```

Comment:
This isn't trimming whitespace, it's removing a blank line which was there for readability so I'd prefer it to stay.